projects/ccf-api/src/lib/routes/browser.ts

```diff
@@ -21,7 +21,7 @@ export const browserRoute: RequestHandler = (_req, res, _next) => {
 };
 
 export const openApiRoute: RequestHandler = (_req, res, _next) => {
-  const apiFile = resolve('ccf-api-spec.yaml');
+  const apiFile = resolve('projects/ccf-openapi/spec.yaml');
```

Comment:
We want to use the original, unprocessed spec here. This is for the browser UI.

Reply:
Whoops, forgot to change this when I moved the spec back to top level